Gemfile

```diff
@@ -7,4 +7,5 @@ gem 'pry'
 gem 'rspec'
 
 gem 'standardrb'
-gem 'webmock'
\ No newline at end of file
+gem 'webmock'
```

Comment:
We see this because of the lack of a newline at the end of the file. Configure your editor to always add them on save, so we don't get this sort of churn.